core/process/queries/QueryManagerRocketmap.php

```diff
@@ -0,0 +1,589 @@
+
+	public function __construct() {
+		parent::__construct();
+	}
+
+	///////////
+	// Tester
+	///////////
+	
+	function testTotalPokemon() {
+		$req = "SELECT COUNT(*) as total FROM pokemon";
+		$result = $this->mysqli->query($req);
+		if (!is_object($result)) {
+			return 1;
+		} else {
+			$data = $result->fetch_object();
+			$total = $data->total;
+	
+			if ($total == 0) {
+				return 2;
+			}
+		}
+		return 0;
+	}
+	
+	function testTotalGyms() {
+		$req = "SELECT COUNT(*) as total FROM gym";
+		$result = $this->mysqli->query($req);
+		if (!is_object($result)) {
+			return 1;
+		} else {
+			$data = $result->fetch_object();
+			$total = $data->total;
+	
+			if ($total == 0) {
+				return 2;
+			}
+		}
+		return 0;
+	}
+	
+	function testTotalPokestops() {
+		$req = "SELECT COUNT(*) as total FROM pokestop";
+		$result = $this->mysqli->query($req);
+		if (!is_object($result)) {
+			return 1;
+		} else {
+			$data = $result->fetch_object();
+			$total = $data->total;
+	
+			if ($total == 0) {
+				return 2;
+			}
+		}
+		return 0;
+	}
+	
+	
+	/////////////
+	// Homepage
+	/////////////
+	
+	function getTotalPokemon() {
+		$req = "SELECT COUNT(*) AS total FROM pokemon WHERE disappear_time >= UTC_TIMESTAMP()";
+		$result = $this->mysqli->query($req);
+		$data = $result->fetch_object();
+		return $data;
+	}
+	
+	function getTotalLures() {
+		$req = "SELECT COUNT(*) AS total FROM pokestop WHERE lure_expiration >= UTC_TIMESTAMP()";
+		$result = $this->mysqli->query($req);
+		$data = $result->fetch_object();
+		return $data;
+	}
+	
+	function getTotalGyms() {
+		$req = "SELECT COUNT(DISTINCT(gym_id)) AS total FROM gym";
+		$result = $this->mysqli->query($req);
+		$data = $result->fetch_object();
+		return $data;
+	}
+	
+	function getTotalRaids() {
+		$req = "SELECT COUNT(*) AS total FROM raid WHERE start <= UTC_TIMESTAMP AND  end >= UTC_TIMESTAMP()";
+		$result = $this->mysqli->query($req);
+		$data = $result->fetch_object();
+		return $data;
+	}
+	
+	function getTotalGymsForTeam($team_id) {
+		$req = "SELECT COUNT(DISTINCT(gym_id)) AS total FROM gym WHERE team_id = '".$team_id."'";
+		$result = $this->mysqli->query($req);
+		$data = $result->fetch_object();
+		return $data;
+	}
+	
+	function getRecentAll() {
+		$req = "SELECT DISTINCT pokemon_id, encounter_id, disappear_time, last_modified, (CONVERT_TZ(disappear_time, '+00:00', '".$this->time_offset."')) AS disappear_time_real,
+					latitude, longitude, cp, individual_attack, individual_defense, individual_stamina
+					FROM pokemon
+					ORDER BY last_modified DESC
+					LIMIT 0,12";
+		$result = $this->mysqli->query($req);
+		$data = array();
+		if ($result->num_rows > 0) {
+			while ($row = $result->fetch_object()) {
+				$data[] = $row;
+			}
+		}
+		return $data;
+	}
+	
+	function getRecentMythic($mythic_pokemons) {
+		$req = "SELECT DISTINCT pokemon_id, encounter_id, disappear_time, last_modified, (CONVERT_TZ(disappear_time, '+00:00', '".$this->time_offset."')) AS disappear_time_real,
```

Comment:
Here too, `$this->time_offset` must be `self::time_offset`

Reply:
fixed